src/translib/transformer/xfmr_ztp.go

```diff
@@ -300,7 +300,7 @@ var DbToYang_ztp_status_xfmr SubTreeXfmrDbToYang = func (inParams XfmrParams) (e
     targetUriPath, err := getYangPathFromUri(pathInfo.Path)
     if targetUriPath == "/openconfig-ztp:ztp/state" {
 	log.Info("TARGET URI PATH ZTP:", targetUriPath)
-        log.Info("TableXfmrFunc - Uri ZTP: ", inParams.uri);
+        xfmrLogInfoAll("TableXfmrFunc - Uri ZTP: %v", inParams.uri);
         log.Info("type of ZTP-ROOT OBJECT:",reflect.TypeOf(ztpObj))
```

Comment:
We initially made changes to the logs where parameters were printed from inParams. But doing this only when we print dbDataMap in inParams only as this can contains huge data.